common/djangoapps/student/views.py

```diff
@@ -25,7 +25,9 @@
 from django.http import (HttpResponse, HttpResponseBadRequest, HttpResponseForbidden,
                          HttpResponseServerError, Http404)
 from django.shortcuts import redirect
+from django.utils import timezone
 from django.utils.translation import ungettext
+from datetime import timedelta
```

Comment:
Please add imports alphabetically and separate **django imports** from third **party imports** and **local imports**